src/components/ranking-chart/ranking-chart.js

```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
 import Component from './ranking-chart-component';
 import mapStateToProps from './ranking-chart-selectors';
-
+import useDebounce from 'hooks/use-debounce';
```

Comment:
We are already using `lodash`'s `debounce` on `countries-borders-layer.js`. Since we are already importing it maybe this `hook` is redundant (`lodash`'s `debounce` has some builtin options like `leading` and `trailing` that are used on the countries borders events so the hook is not ready to be used there).
One thing that might be needed is importing it as `import debounce from 'lodash/debounce'` instead of `import { debounce } from 'lodash'` as we are currently doing.
What do you think?

Reply:
lodash debounce in this case it's not easy to use. We could find another workaround but the problem is that debounce just returns a function and cant accept arguments on its contained function